hummingbird/ml/operator_converters/sklearn/poly_features.py

```diff
@@ -74,8 +72,8 @@ def convert_sklearn_poly_features(operator, device, extra_config):
     if operator.raw_operator.interaction_only:
         raise NotImplementedError("Hummingbird does not currently support interaction_only flag for PolynomialFeatures")
 
-    if operator.raw_operator.degree != 2:
-        raise NotImplementedError("Hummingbird currently only supports degree 2 for PolynomialFeatures")
+    if operator.raw_operator.degree < 0:
+        raise NotImplementedError("Hummingbird not supports negtive degree for PolynomialFeatures")
```

Comment:
Can you please change to "Hummingbird does not support negative degree for PolynomialFeatures"